package.json

```diff
@@ -10,10 +10,10 @@
     "build": "tsc --pretty",
     "buildw": "tsc -w --pretty",
     "testw": "mocha --recursive --watch build/test",
-    "pretest": "tsc --pretty && mocha --recursive --reporter progress build/test",
-    "test": "tslint ./src/*.ts ./test/*.ts",
+    "pretest": "npm run build",
+    "test": "mocha --recursive --reporter progress build/test",
     "lint": "tslint ./src/*.ts ./test/*.ts",
-    "posttest": "browserify build/src/web.js -o web/bundle.js"
+    "browserify": "browserify build/src/web.js -o web/bundle.js"
```

Comment:
browserify will not be run any longer?

Reply:
well, now it's `npm run browserify` :) But this one is arguable, could you please tell what is the further flow with it ? What I meant is that browserify does not belong to test flow imho, so maybe it should be a part of another process.

Reply:
for this one waiting for your reply